CONTRIBUTING.md

```diff
@@ -1 +1,4 @@
-Please see [Contribution Guidelines](https://developers.mattermost.com/contribute/getting-started/)
+
+## Contribute to Mattermost (https://mattermost.com/contribute/)
```

Comment:
Why is this headline place first?  Should the information about this specific repository i.e. the link to https://developers.mattermost.com/contribute/getting-started/ be placed first?

Reply:
The first link is to the landing page which lists all the places/areas where you can contribute. The second link is the specific steps to follow for guidelines. Because this is the docs repo (not dev docs) I think the placement does work. If it was a developer repo I'd definitely change it.

Reply:
I totally forgot this is the docs repo. Thanks for reminding me :+1: